grc/core/blocks/dummy.py

```diff
@@ -28,9 +29,12 @@ class DummyBlock(Block):
     label = 'Missing Block'
     key = '_dummy'
 
-    def __init__(self, parent, missing_block_id, parameters, **_):
+    def __init__(self, parent, missing_block_id, parameters,**kwargs):
+
         self.key = missing_block_id
+        self.parameters_data = _build_params([kwargs],False, False,self.flags, self.key)
```

Comment:
What are you trying to do here exactly? I can see the advantage of adding the the standard params with this call (known type, category, ...). What I don't get is why you need to pass in kwargs? What is this supposed to be doing? 
Since the code in line 40 is supposed to be doing just that, I believe, an empty list should be passed here

Reply:
...or properly replace that code by constructing params_raw appropriately

Reply:
@skoslowski You are right, kwargs is not needed. An empty list does the trick and I don't have to remove self.params[self.key]

But I wonder if this piece of code should be part of the block's __init__ function, as the generated parameters belong  to a core part of the block and each sublcassed block.